src/pages/hiring-process/adapter/hiring-process-adapter.js

```diff
@@ -1,4 +1,7 @@
 export const hiringProcessAdapter = (hiringProcesses) => {
+  const getPropertyFromExercises = (exercises, property) => (
+    exercises.map(exercise => exercise.evaluation[property] || ''))
```

Comment:
Um ponto de atenção aqui é que usar `||` pode gerar comportamentos inesperados dependendo da situação, pois `||` será falso se o primeiro operando for "falsy" (o que inclui valores como `0`, `""`, `false`, `[]`)  , não somente `null` ou `undefined`. Se quisermos que a operação seja false somente para `null` ou `undefined`, devemos usar o null-coalescing operator:

```suggestion
    exercises.map(exercise => exercise.evaluation[property] ?? ''))
```

Documentação sobre null-coalescing operator:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

Reply:
Esse operador não é exclusivo do Javascript, em algumas outras linguagens (Kotlin, por exemplo) ele é conhecido como [Elvis Operator](https://en.wikipedia.org/wiki/Elvis_operator)(?:).